src/main/java/eu/dissco/core/translator/terms/DwcaDigitalObjectDirector.java

```diff
@@ -51,8 +51,12 @@ protected List<Citation> assembleSpecimenCitations(JsonNode data, boolean dwc) {
         && data.get(EXTENSION).get("gbif:Reference") != null) {
       var references = data.get(EXTENSION).get("gbif:Reference");
       for (int i = 0; i < references.size(); i++) {
-        var identification = references.get(i);
-        citations.add(createCitation(identification, dwc));
+        var citationJson = references.get(i);
+        if (citationJson.properties().size() <= 1){
+          log.debug("Skipping citation with only one property: {}", citationJson);
```

Comment:
Can there be citations with 0 properties? If yes, maybe change the log to "..with 1 or fewer properties".

Reply:
No, if we create a citationObject because we encountered the dwca extension, we always add the type. This meant that even when there wasn't a citation for the specimen, we got citations with only the @type in there. This looked ugly and isn't very helpful, so now if we encounter a citation with only just one property (the @type) we ignore it.